syntax = "proto3";
package rpc_package;

service RecoService {
    rpc SayReco (RecoRequest) returns (RecoReply) {}
}

message RecoRequest {
    repeated Video video = 1;
}

message Video{
    int32 age = 1;
    string name = 2;
}

message RecoReply {
    string message = 1;
    repeated double score_list = 2;
}